plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout.spec.js

```diff
@@ -364,8 +381,10 @@ test.describe( 'Checkout page', () => {
 		await expect( page.locator( 'strong.product-quantity' ) ).toContainText(
 			'2'
 		);
-		await expect( page.locator( 'td.product-total' ) ).toContainText(
-			twoProductPrice
+		let totalPrice = await page.getByRole( 'row', { name: 'Total' } ).last().locator( 'td' ).textContent();
+		totalPrice = Number( totalPrice.replace( /[^\d.-]/g, '' ) );
+		await expect( totalPrice ).toBeGreaterThanOrEqual(
+			Number( twoProductPrice )
```

Comment:
I changed up the assertion on these.

For the purposes of these tests, we're okay with just making sure that the order total is at least the price of one/two/three products.  If there are taxes applied or something like that, the total might be higher.